shell/run_shell_cmd_unix_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux || darwin
```

Comment:
I see that sometimes you use `!windows` instead of `linux || darwin`, is there a reason?

Reply:
🤷